lawnchair/src/app/lawnchair/preferences/PreferenceManager.kt

```diff
@@ -51,7 +51,8 @@ class PreferenceManager private constructor(private val context: Context) : Base
 
     val drawerOpacity = FloatPref("pref_drawerOpacity", 1F, recreate)
     val coloredBackgroundLightness = FloatPref("pref_coloredBackgroundLightness", 0.9F, recreate)
-    val feedProvider = StringPref("pref_feedProvider", "")
+    val feedProvider = StringPref("pref_feedProvider", "", recreate)
```

Comment:
Maybe we can add restart here? or take a warning when switching to feed may need lawnchair to restart?

Reply:
I think we should make it apply without restarting instead. That would be a better user experience.

Reply:
I think we should make it apply without restarting instead. That would be a better user experience.